docs/sources/auth/planning/index.md

```diff
@@ -85,7 +85,38 @@ Consider the following scenarios:
 
 These are just a few examples of how Grafana can be used in M2M scenarios. The platform is highly flexible and can be used in a variety of other M2M applications, making it a powerful tool for organizations looking to gain insights into their systems and devices.
 
-### 🚧 Service Accounts
+### Service accounts
+
+You can use a service account to run automated workloads in Grafana, such as dashboard provisioning, configuration, or report generation. Create service accounts and service accounts tokens to authenticate applications, such as Terraform, with the Grafana API.
+
+> **Note:** Service accounts will eventually replace [API keys]({{< relref "../api-keys/" >}}) as the primary way to authenticate applications that interact with Grafana.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control/" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+> **Note:** Service accounts can only act in the organization they are created for. If you have the same task that is needed for multiple organizations, we recommend creating service accounts in each organization.
+
+{{< vimeo 742056367 >}}
+<br>
+
+## Service account tokens
+
+A service account token is a generated random string that acts as an alternative to a password when authenticating with Grafana's HTTP API.
```

Comment:
prefixed with `glsa` , I feel this information is important so people can recognize a token easily

Reply:
Agreed! However, keep in mind this section is only about a brief description of Service account tokens during planning. I'll make sure to include this during the reference development for SAT